handle non-treats queries, produce assets

sierra-moxon opened the pull request.

maximusunc commented on test_assets/Asset_135.json:
```diff
@@ -0,0 +1,40 @@
+    "input_id": "NCBI Gene:8653",
+    "input_name": "DDX3Y",
+    "input_category": null,
+    "predicate_id": "biolink:biolink:decreases_activity_or_abundance_of",
+    "predicate_name": "biolink:decreases_activity_or_abundance_of",
```
These have an extra biolink added

sierra-moxon replied:
ok I stripped the biolink - it would be good to sync up on how we want to pass the predicates - I guess you add these prefixes in the harness?

maximusunc commented on test_assets/Asset_135.json:
```diff
@@ -0,0 +1,40 @@
+        "test_objective": "AcceptanceTest",
+        "test_annotations": []
+    },
+    "biolink_qualified_predicate": null,
+    "biolink_subject_aspect_qualifier": null,
+    "biolink_subject_direction_qualifier": null,
+    "biolink_object_aspect_qualifier": null,
+    "biolink_object_direction_qualifier": null
```
Should these be filled out?

sierra-moxon replied:
yep, done.

maximusunc commented on test_assets/Asset_307.json:
```diff
@@ -0,0 +1,40 @@
+{
+    "id": "Asset_307",
+    "name": "Albuterol_increases activity or abundance of _Beta_2_adrenergic_(ADRB2)",
+    "description": "Albuterol_increases activity or abundance of _Beta_2_adrenergic_(ADRB2)",
+    "tags": [],
+    "input_id": "NCBIGene:154",
+    "input_name": "Beta 2 adrenergic (ADRB2)",
+    "input_category": null,
+    "predicate_id": "biolink:treats",
+    "predicate_name": "treats",
```
This name and predicate doesn't match.

maximusunc commented on test_assets/Asset_308.json:
```diff
@@ -0,0 +1,40 @@
+{
+    "id": "Asset_308",
+    "name": "Propranalol_increases activity or abundance of _Beta_2_adrenergic_(ADRB2)",
+    "description": "Propranalol_increases activity or abundance of _Beta_2_adrenergic_(ADRB2)",
+    "tags": [],
+    "input_id": "NCBIGene:154",
+    "input_name": "Beta 2 adrenergic (ADRB2)",
+    "input_category": null,
+    "predicate_id": "biolink:treats",
+    "predicate_name": "treats",
```
This name and predicate don't match

maximusunc commented on test_assets/Asset_313.json:
```diff
@@ -0,0 +1,40 @@
+{
+    "id": "Asset_313",
+    "name": "Ivacaftor_increases activity or abundance of _CFTR_(human)",
+    "description": "Ivacaftor_increases activity or abundance of _CFTR_(human)",
+    "tags": [],
+    "input_id": "NCBIGene:1080",
+    "input_name": "CFTR (human)",
+    "input_category": null,
+    "predicate_id": "biolink:biolink:decreases_activity_or_abundance_of",
+    "predicate_name": "biolink:decreases_activity_or_abundance_of",
```
These don't match

maximusunc commented on test_cases/TestCase_0.json:
```diff
@@ -1,23 +1,23 @@
     "id": "TestCase_0",
-    "name": "what treats MONDO:0001119",
-    "description": "Basis_treats_Premature_Menopause; Hexadrin_treats_Premature_Menopause; Busulfan_treats_Premature_Menopause; Lasofoxifene_treats_Premature_Menopause; Busulfan_treats_Premature_Menopause; estradiol_treats_Premature_Menopause; progestin_treats_Premature_Menopause",
+    "name": "what treats MONDO:0011426",
```
Is there a way for Test Cases to persist? It seems like the Test Cases are getting overwritten each time, which just makes it hard to tell any diffs.

sierra-moxon replied:
it will (hopefully) once be easier when we have DH in place.  right now the spreadsheet is changing almost every time I parse it.

sierra-moxon commented:
@maximusunc - I changed a bunch of stuff here - stripped whitespace, fixed up the new properties to not be null, re-matched predicates with their names, and updated the assets with a fresh pull of the spreadsheet.

maximusunc commented on test_assets/Asset_1.json:
```diff
@@ -1,15 +1,15 @@
-    "input_name": "Aceruloplasminemia",
-    "input_category": null,
-    "predicate_id": "biolink:treats",
+    "input_name": "MONDO:0011426",
```
Input name isn't getting set correctly.

maximusunc commented on test_assets/Asset_1.json:
```diff
@@ -1,15 +1,15 @@
-    "predicate_id": "biolink:treats",
+    "input_name": "MONDO:0011426",
+    "input_category": "biolink:Disease",
+    "predicate_id": "treats",
```
Should this include the `biolink:` prefix?

sierra-moxon replied:
in the last review, you asked me to remove the biolink prefix :) -- for input_id, output_id, predicate, qualified_predicate -should we have the biolink prefix or not?

maximusunc replied:
The issue before was that some predicate_ids had this: `biolink:biolink:<predicate>`, and then the predicate name also had `biolink:<predicate>`. Now neither predicate_id nor predicate_name have the biolink prefix.

maximusunc replied:
For your question about the other fields, yes, I think those should have the biolink prefix.

maximusunc commented:
With this PR, could you also make a new release of the TranslatorTestingModel please?

maximusunc reviewed (approved):
Ran some tests and looks good!
